Targets/UserInterface/Sources/Home/HomeViewModel.swift

```diff
@@ -48,15 +48,17 @@ final class HomeViewModel: ObservableObject {
     private var anyCancellables: [AnyCancellable] = []
     
     @Published var homeRows: [[HomeTag]] = []
-    @Published var homeTags: [HomeTag] = [ ]
+    @Published var homeTags: [HomeTag] = []
     
     init(modelContainer: ModelContainer) {
         self.modelContainer = modelContainer
         self.modelContainer.tagModel.itemsPublisher
             .receive(on: RunLoop.main)
-            .sink { tags in
-                    self.homeTags = [HomeTag(tag: WholeTag())]
-                    self.homeTags.append(contentsOf: tags.map { HomeTag(tag: $0 )})
+            .sink { [weak self] tags in
+                self?.homeTags = [HomeTag(tag: WholeTag())]
+                self?.getTags(tags.map{ tag in
+                    HomeTag(tag: tag)
+                })
             }
```

Comment:
전체 태그인 WholeTag가 포함되지 않은 채, getTags 메소드에 보내지고 있습니다!
```suggestion
            .sink { tags in
                self.homeTags = [HomeTag(tag: WholeTag())]
                self.homeTags.append(contentsOf: tags.map { HomeTag(tag: $0 )})
                self.getTags(self.homeTags)
            }
```
이렇게 수정 제안 드립니다!